src/encoded/static/scss/encoded/_base.scss

```diff
@@ -257,6 +257,11 @@ $disabled-color-factor: 20%;
 	}
 }
 
+input[disabled],
+button[disabled] {
+    cursor: not-allowed;
+}
```

Comment:
Looks good! Just a question: it could also work to write `input: disabled, button: disabled` -- is it generally the Encode style to use the brackets instead?

Reply:
I don’t think we have a set style for this.